packages/@react-stately/virtualizer/src/ReusableView.ts

```diff
@@ -31,7 +31,7 @@ export class ReusableView<T extends object, V> {
   layoutInfo: LayoutInfo | null;
 
   /** The content currently being displayed by this view, set by the collection view. */
-  content: T;
+  content: any;
```

Comment:
Curious why this and below switched to `any`?

Reply:
In Virtualizer I'm checking content.props.allowsResizing. This was throwing type errors because props doesn't exist on type `T`. I spent some time trying to figure out what I need to do to get it to work correctly and be typed right but wasn't having any luck and needed to work on some other stuff so I set it to `any`. This is the commit with the change: [1999421](https://github.com/adobe/react-spectrum/pull/2883/commits/1999421f0fa2dd42e6df2cbdebd1aaab2567643c)

I am open to suggestions on how to do the typing correctly so we can avoid using `any`